proxy/http/HttpConfig.cc

```diff
@@ -1129,6 +1129,8 @@ HttpConfig::startup()
   // Local Manager
   HttpEstablishStaticConfigLongLong(c.synthetic_port, "proxy.config.admin.synthetic_port");
 
+  // FTP protocol enabled
+  HttpEstablishStaticConfigByte(c.forward_proxy_ftp_enabled, "proxy.config.ftp_enabled");
```

Comment:
Do we need to change this? If so, could you provide some more details on what you want us to do.